### DISCUSSION TITLE
postCreate doesn't work for github codespaces when launching from vscode desktop

### DISCUSSION THREAD

**Discussion by nojvek** [community/community#5652] [Codespaces]
Expected:

The `postStart: "yarn install"` command is run when vscode remote is connected.

Actual:

Doesn't run. I couldn't get either `postCreate` or `postStart` to correctly run.

It's finicky. I would assume it would show some sort of a log, but it doesn't

**Comment by Chuxel** [comment]
Do you mean `postCreateCommand`? There is no `postCreate` property.
